src/main/java/oshi/hardware/platform/windows/WindowsGlobalMemory.java

```diff
@@ -68,6 +69,18 @@ public long getTotal() {
     }
 
     @Override
+    public long getSwapTotal() {
+        // TODO: for now, it returns 0

```

Comment:
Windows swap info is part of the [MEMORYSTATUSEX](https://msdn.microsoft.com/en-us/library/windows/desktop/aa366770(v=vs.85).aspx) structure.  Copy code from `getAvailable()` and change which structure element you're returning.